### PR TITLE
2 get api

### PR DESCRIPTION
**Pull request by wttocs**
Forgot to branch off initial task on trello for readme

Pull request is for the completion of task 3 - get/api/topics with TDD

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+require("jest-sorted");
```
Comment (P-Copley) [inline __tests__/app.test.js:6]: Any reason why you've required this in as you don't use it?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
```
Comment (P-Copley) [inline __tests__/app.test.js:13]: You should return this promise. Your current setup shouldn't run into any issues but by returning it jest can wait for it to resolve before moving on.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+});
+
+// Test to check tests are running
+describe("App tests are running", () => {
+  test("app.tests is running...", () => {});
+});
```
Comment (P-Copley) [inline __tests__/app.test.js:19]: I reckon you can take these out now 😁

**Comment by P-Copley** [comment]
Excellent start, the endpoint setup and testing looks great 😁  

Once you've had a look at the above feel free to merge 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (wttocs) [reply __tests__/app.test.js:6]: Just out of habit, will remove :)

**Pull request merged**